### PR TITLE
Angular add Build Target for Web

### LINKED ISSUE #539: Add web-only frontend code inspired by webapp frontend hacks
blocks #467 

- [x] Add build target in angular.json
- [x] Create custom environment.web.ts
- [x] Bootstrap services
- [x] Modify Dependency Injections of Services in Components
- [x] Implement Services for Web Build Target (Blocked by #649 )

### PR DESCRIPTION
**Pull request by Epsilon02**
part of #539

### CODE REVIEW

**Comment by Epsilon02** [comment]
comment to deactivate stale

**Comment by Epsilon02** [comment]
The Application will now use the correct service depending of the build target.
@crapStone it is now your turn to implement the services.

File: angular.json [hunk 1]
```diff
@@ -68,6 +62,14 @@
                   "maximumError": "4kb"
                 }
               ]
+            },
+            "web": {
+              "fileReplacements": [
+                {
+                  "replace": "src/environments/environment.ts",
+                  "with": "src/environments/environment.prod.ts"
```
Comment (6543) [inline angular.json:70]:
```suggestion
                  "with": "src/environments/environment.web.ts"
```

**Comment by 6543** [comment]
can you also add a `start:web`

**Review by 6543 — DISMISSED** [review]

**Review by 6543 — APPROVED** [review]

**Pull request merged**